spec/integration/kaminari_spec.rb

```diff
@@ -37,7 +37,7 @@
       content[:response].should be_present
       content[:count].should == 5
       content[:pagination].should be_present
-      content[:pagination][:count].should == 5
+      content[:pagination][:count].should == 25
```

Comment:
Must be from the kaminari bump. Maybe its not actually a semver "minor". I guess I'm not positive that this version bump propagates up to monorail (since its not in monorail Gemfile.lock) but I see how the specs would need to be updated this way.